dragonfly/implementations/dungbeetle.py

```diff
@@ -33,9 +33,22 @@ def __init__(self,
         self.processed_dirs = {}
         self.warning_interval = warning_interval
 
+    # delete old .nfs files that get stranded 
+    def clean_nfs(self, path, min_creation_time):
+        creation_time = datetime.datetime.fromtimestamp(os.path.getctime(path))
+        if creation_time < min_creation_time and (not path in self.ignore_dirs):
+            try:
+                os.remove(path)
+                logger.info("nfs file [{}] has been removed.".format(path))
+            except:
+                pass

```

Comment:
We should be more precise in the exception we're catching here and what we do in such a case. Probably `OSError` like below and then a critical alert that prints the message from that error so a human is notified to investigate.